### ISSUE TITLE
Cannot install via `gcloud components install` on Apple Silicon (M1) 

### ISSUE THREAD

**Issue by takamario** [GoogleCloudPlatform/cloudsql-proxy#938]
<!--

Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an 
issue in the [support console](https://cloud.google.com/support/) instead of 
filing on GitHub. This will ensure a timely response.

Please run down the following list and make sure you've tried the usual "quick fixes":

  - Search the issues already opened: https://github.com/GoogleCloudPlatform/cloudsql-proxy/issues
  - Check for answers on StackOverflow: https://stackoverflow.com/questions/tagged/google-cloud-sql

If you are still having issues, please include as much information as possible:

--> 

## Bug Description

Cannot install via `gcloud components install` on Apple Silicon (M1) Mac even though it has arm64 binary.

## Example code (or command)

I tried the following and already `rosetta` installed.
```sh
$ gcloud components install cloud_sql_proxy                                                                                                                                                                                                                       
WARNING: The ARM versions of the components [cloud_sql_proxy] are not available yet. To download and execute the x86_64 version of the components, please install Rosetta 2 first by running the command: softwareupdate --install-rosetta.
WARNING: The ARM versions of the components [cloud_sql_proxy] are not available yet. To download and execute the x86_64 version of the components, please install Rosetta 2 first by running the command: softwareupdate --install-rosetta.
ERROR: (gcloud.components.install) The following components are unknown [cloud_sql_proxy].
```

I could install other components (e.g. `kubectl`, `beta`) via `gcloud components install` 

## Environment

1. OS: macOS BigSur 11.5.2
2. chip: Apple M1  
3. gcloud: 357.0.0

**Comment by enocom** [comment]
@takamario Thanks for the report. As a work-around, we do provide an M1 build on the [releases page](https://github.com/GoogleCloudPlatform/cloudsql-proxy/releases).

**Comment by enocom** [comment]
The gcloud version of the proxy is currently 1.18.0, which is several minor versions behind latest (1.25.0). We recommend obtaining the latest from the releases page.

**Comment by takamario** [comment]
Got it about latest version installation.

Same as https://github.com/GoogleCloudPlatform/cloudsql-proxy/issues/408, we would be happy if gcloud could update to 1.20.0+ to support arm64 binary.

**Comment by enocom** [comment]
I've found the internal owner of this work and they'll get an update out soon. 

As a side note, we're looking at automating this process, but that work is currently blocked on some internal technical details.

**Comment by enocom** [comment]
The internal change to support this is working its way through review.

**Comment by enocom** [comment]
That's surprising. As long as your PATH has been updated to work with the gcloud SDK, the `cloud_sql_proxy` should be on the path as well at `$PATH_TO_SDK/bin/cloud_sql_proxy`.

Separately, the internal change to update the gcloud proxy component to latest (v1.27.0) with M1 support has been merged. I don't know what version that change will land in, but I'd it expect it to be available soon.

**Comment by mattrussell-sonocent** [comment]
> That's surprising. As long as your PATH has been updated to work with the gcloud SDK, the `cloud_sql_proxy` should be on the path as well at `$PATH_TO_SDK/bin/cloud_sql_proxy`.

Ah, I hadn't done a `gcloud init`, apologies for the noise.

> Separately, the internal change to update the gcloud proxy component to latest (v1.27.0) with M1 support has been merged. I don't know what version that change will land in, but I'd it expect it to be available soon.

Great, thanks for the update 👍

**Comment by enocom** [comment]
v1.27.0 (one patch release behind latest) is now available in gcloud 367.0.0, which includes M1 support.
